cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -451,6 +457,15 @@ public boolean verify(String s, SSLSession sslSession) {
                 args = args.subList(2,args.size());
                 continue;
             }
+            if(head.equals("-v")) {
+                args = args.subList(1,args.size());
+                Logger l = Logger.getLogger(ROOT_LOGGER_NAME);
+                l.setLevel(FINE);
+                for (Handler h : l.getHandlers()) {
+                    h.setLevel(FINE);
```

Comment:
I feel it is a 🐛 , because this setting may *reduce* the logging level, e.g. if it is configured as DEBUG by the j.u.l. property file